tests/unit/fixtures/good_config_json.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Are we using this file in this PR? Can we remove it for now and add it back when its needed ?

Reply:
It's used in the test_config.py file: https://github.com/DataDog/apigentools/blob/a222c9dcaaf1b83533ae936f0f1a5dd9737f230a/tests/test_config.py#L7-L8

If I did move it over, the CI test for `test_config_from_file` in the test_config file will fail. Do you think it's worth merging the PR with a failing test?

I did move the fixtures/ directory from the top-level tests/ directory to sit inside the unit/ directory, but the file itself didn't change.